hpms/src/test/java/gov/cms/ab2d/hpms/service/HPMSAuthServiceTest.java

```diff
@@ -22,6 +22,9 @@
     @Autowired
     HPMSAuthServiceImpl authService;
 
+    @Autowired
+    HPMSFetcherImpl hpmsFetcher;
+
```

Comment:
Don't see a need for this especially in the AuthService which should be standalone.